spec/spec_helper.rb

```diff
@@ -1,10 +1,22 @@
 require 'nest_thermostat'
 require 'dotenv'
 require 'pry'
+require "fakeweb"
+require 'vcr'
+require 'timecop'
+require 'awesome_print'
 
 Dotenv.load
 
 RSpec.configure do |c|
   c.filter_run focus: true
   c.run_all_when_everything_filtered = true
+  c.extend VCR::RSpec::Macros
+end
+
+VCR.configure do |c|
+  c.cassette_library_dir = 'fixtures/vcr_cassettes'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.